### ISSUE TITLE
Mongo service on Ubuntu 16.04 EC2 AWS

### ISSUE THREAD

**Issue by ht2** [LearningLocker/deploy#14]
Need to double check that the Mongo actually starts prior to running the script. Sneaky suspicion that it doesn't

**Comment by gardron** [comment]
Fixed in latest push - the mongo repo on 16.04 creates a slight issue which needs resolving (unmasking startup scripts) before the service can start

**Comment by ht2** [comment]
Seems to be a sneaky issue here with the name of the service still:

```
[LL] Setting up mongo repo (Stock Ubuntu version is too old)
Executing: /tmp/tmp.q6mRBWe1uQ/gpg.1.sh --keyserver
hkp://keyserver.ubuntu.com:80
--recv
0C49F3730359A14518585931BC711F9BA15703C6
gpg: requesting key A15703C6 from hkp server keyserver.ubuntu.com
gpg: key A15703C6: public key "MongoDB 3.4 Release Signing Key <packaging@mongodb.com>" imported
gpg: Total number processed: 1
gpg:               imported: 1  (RSA: 1)
deb [ arch=amd64,arm64 ] http://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/3.4 multiverse

Failed to start mongodb.service: Unit mongodb.service not found.
```

This can be addressed by either starting the `mongod` service (which _did_ seem to exist for me) or creating a new `mongodb` service. This SO answer had a working one for me: https://stackoverflow.com/a/37014253
